corehq/apps/export/models/new.py

```diff
@@ -935,6 +935,7 @@ def put_item(self, path):
         item = ExportItem(
             path=path,
             label='.'.join(map(lambda node: node.name, path)),
+            last_occurrences={INFERRED_OCCURRENCE: INFERRED_OCCURRENCE}
```

Comment:
The first alternative that comes to mind is to add an `is_inferred_occurrence` flag to `ExportItem` instead. It makes things a little more verbose, but I think it is more obvious whats going on than having to know the format of the special `last_occurrences` values

If you don't do that, I think its worth adding an `is_inferred_occurrence` property to `ExportItem` as a way of sort of "documenting" what the special `last_occurrences` property means.